package.json

```diff
@@ -144,6 +144,7 @@
     "bluebird": "3.5.5",
     "boom": "^7.2.0",
     "brace": "0.11.1",
+    "broadcast-channel": "^2.3.2",
```

Comment:
This module is pretty well-maintained, and it didn't look like any of the dependencies were alarming or ones that we'd otherwise want to avoid.